src/sentry/static/sentry/app/components/asyncComponent.tsx

```diff
@@ -403,6 +407,10 @@ export default class AsyncComponent<
   renderError(error?: Error, disableLog = false, disableReport = false): React.ReactNode {
     const {errors} = this.state;
 
+    if (this.shouldRaiseRequestErrorToParent) {
+      throw error;
+    }
```

Comment:
Why not have your component override `renderError`?

Reply:
@billyvg we get a type error because the function returns `void`

Reply:
Can we return `null`?

Reply:
@billyvg i think i found a workaround by explicitly setting the return type to `ReactNode` for `renderError`